tutorials/_Quick_start.md

```diff
@@ -0,0 +1,78 @@
+## !! Renv !!
+Note that REMIND **releases do not come with a locked environment** ([see the associated issue, which may be closed by the time you start](https://github.com/remindmodel/development_issues/issues/528)). This means that you will need to restore the version corresponding to your release: for example `make historic-reqs v=2024-12-11` where the date should be replaced with that of the release (see [here](https://github.com/remindmodel/development_issues/issues/528)) 
+
+# 3. Your first run
+*Note <some_text> indicate place holders*
+*To learn about the HPC and slurm have a look at [the IT HPC guide](https://www.pik-potsdam.de/en/institute/about/it-services/hpc/hpc-2024/hpc-2024-documentation/hpc-2024-documentation)* (you can use remind without knowing anything about high performance computers or the slurm job manager)
+
+- make a copy of `config/scenario_config.csv` -> `config/<myfirstrun_cfg>.csv` 
+	-  keep only 2-3 rows, say those in table1
+	-  **NB: the config uses German delimiters `;`**
+	- `scenario_config.csv` represents all the possible scenarios. The others don't really belong in a code repo but are there
+	- How the config.csv work is explained in [the REMIND tutorial 03](https://github.com/remindmodel/remind/blob/develop/tutorials/03_RunningBundleOfRuns.md)
+	- do not edit `default.cfg` which is actually an R script
+- check your run
+	- `Rscript start.R --test config/<myfirstrun_cfg>.csv`
+	- press enter to (mock) submit to the priority queue
+- launch your run
+	- `Rscript start.R config/<myfirstrun_cfg>.csv`
+	- press enter to submit to the priority queue 
+
+| Name            | Descriptor        |
+| --------------- | ----------------- |
+| SSP2-NPi2025    | Current policies  |
+| SSP2-PkBudg500  | 1.5C no overshoot |
+| SSP2-PkBudg1000 | 2.C               |
+
+# 4. Check the status of your run
+
+- in a bash terminal on the hpc (e.g. in vscode) type  `squeue -u $USER` your job should appear there
+	- you can also define a more power alias such as `alias sq='squeue -u $USER -o "%.18i %.9P %q %.30j %.8u %.5T %.10M %.9l %.6D %R"'` in your `.bashrc` and run that
+	- if needed you can also check the state of the hpc with `sinfo` and `sacct` commands
+- you should get an email once your run finishes
+	- remind runs in about 3 hours
+
+>[!warning] Failed runs with release versions
+> note that remind releases are not currently shipped with an exact (locked) environment and you may therefore run into dependency issues.
+
+# 5. Look at your results
+
+## Generating a PDF
+*This should be your first step*. If you are at PIK, have a look at [compareScenarios in remind2](https://pik-piam.r-universe.dev/articles/remind2/compareScenariosRemind2.html)
+
+In short, from your REMIND folder run
+- `Rscript output.R`
+- Select comparison across runs: `2: Comparison across runs`
+- and then `compareScenarios2`
+- submit to priority (Not sure how to deal with this if you are not on cluster)
```

Comment:
if you are not on the cluster, it will just execute it directly in your `R` session.